packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy.scss

```diff
@@ -104,8 +101,10 @@
             }
         }
         &-content {
-            margin-bottom: 12rem;
-            padding: 0 1.6rem;
+            margin: 0 1.6rem 0 4.8rem;
+            @include mobile {
+                margin-left: 1.6rem;
```

Comment:
I think rem is not working properly in mobile mode ... could you plz double check?

Reply:
@sara-fs when I did it for the mobile version of course I checked it😁 but if you have any doubts - ok, I will take attention to this

Reply:
@sara-fs thank you for take attention to this detail, I've just checked. As I see we have got styles issues, but it is related to the header for all tabs in the mobile version, it will be done later i suppose

Reply:
![Uploading Screenshot 2022-09-29 at 14.19.45.png…]()

Reply:
@maryiafrantsava-binary  changing the unit  to `px` doesn't help fixing the ui issue?

Reply:
@prince-deriv actually this rem works fine
![Screenshot 2022-09-30 at 11 23 39](https://user-images.githubusercontent.com/103181650/193226536-5f0e1dbb-5ed2-4058-af17-de631a2b8486.png)